### PR TITLE
Featured channels, windows dual-booting, languages, and TOCs

### PR DESCRIPTION
**Pull request by bobheadxi**
# Related Tickets

for #7 

# Changes

* Added TOC to a bunch of pages
* Added @ginsstaahh 's Windows Dual-Booting guide
* Added featured slack channels
* Added a new page, `Languages`, for language-specific resources
    * Added Go resources

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "markdown.extension.toc.levels": "2..6",
```
Comment (rwblickhan) [inline .vscode/settings.json]: Should thisfile be added to .gitignore?

File: GettingStarted/Tools.md [hunk 2]
```diff
@@ -38,6 +49,18 @@ You should also install Slack on your laptop and phone, turn on @mention
 notifications, and respond to messages promptly. Slack has apps available for
 all major platforms.
 
+#### Cool Channels
+
+* `#programming-languages`: General discussion of programming languages and
+  related topics
+* `#graphics`: General discussion of computer graphics and animation
+* `#memes`: Name is self explanatory
```
Comment (rwblickhan) [inline GettingStarted/Tools.md]: This was kinda snarky... might want to  rewrite this entry a bit.

File: .vscode/settings.json (hunk 1 above)
Comment (bobheadxi) [reply .vscode/settings.json]: Not everyone uses VSCode but if you do, without the markdown extension, it'll mess up everything without this configuration 😢 on the bright side, it's actually a pretty handy extension and TOC generation is quite nice, so I'd like to keep this here if thats ok 😋

Comment (rwblickhan) [reply .vscode/settings.json]: Er, what do you mean "mess up"? I'm just more concerned with messing with VSCode settings for people 😛

File: GettingStarted/Tools.md (hunk 2 above)
Comment (bobheadxi) [reply GettingStarted/Tools.md]: 8aac11c ?

Comment (rwblickhan) [reply GettingStarted/Tools.md]: Haha sure, resolving.

File: .vscode/settings.json (hunk 1 above)
Comment (bobheadxi) [reply .vscode/settings.json]: > Er, what do you mean "mess up"?

by default, it generates TOC with `-` and includes top-level `#` headers that we don't want (for us, they're page titles), and does it on save automatically

>  I'm just more concerned with messing with VSCode settings for people

this is only a workspace setting, so it won't do anything outside of `docs`

Comment (rwblickhan) [reply .vscode/settings.json]: Okay fair enough.

**Pull request merged**